### ISSUE TITLE
support podman

### ISSUE THREAD

**Issue by pythops** [robertpsoane/ducker#9]
Would be great to add support for podman as well

**Comment by robertpsoane** [comment]
Definitely makes sense to add, but I'm not sure podman is daemonised in the same way docker is?

I know at the cli api level they're roughly API compatible, but I'm not sure if Bollard is inherently podman compatible

I'll definitely look into it once I've got a few more features on the docker side

**Comment by pythops** [comment]
podman is indeed a  daemonless which makes sense if you think about it, why do you need a daemon to run containers ?
and it has an API that is accessible via unix socket
https://docs.podman.io/en/latest/_static/api.html

**Comment by blurayne** [comment]
Usually a podman installation comes with a socket service (either rootless or root) that was designed to replace dockerd socket. For 99% of all containers / docker-compose stuff you don't have any problems with it. I even added a docker context for it (running docker root/docker rootless/podman on my system). Preference goes to Podman but some devs still rely on docker root :(

https://docs.podman.io/en/v4.1.1/markdown/podman-system-service.1.html

Now you just need to add a pods view. And secrets.

**Comment by robertpsoane** [comment]
I will have a re-read of the docs
My mis-understanding from a brief read about the podman socket service was that there were security implications of it
Thanks for the corrections!

**Comment by DarkAngelStrike** [comment]
+1

**Comment by robertpsoane** [comment]
Revisiting this, my understanding is that setting
`docker_path: unix:///var/run/podman/podman.sock` in the config (`~/.config/ducker/config.yaml`) should be sufficient for it to work with podman?

**Comment by robertpsoane** [comment]
Closing as I believe it should currently support podman

**Comment by DanWlker** [comment]
I dont think this works on macos, which uses something like `ssh://root@127.0.0.1:64299/run/podman/podman.sock`
